data-prepper-plugins/sqs-source/src/main/java/org/opensearch/dataprepper/plugins/source/sqs/QueueConfig.java

```diff
@@ -0,0 +1,92 @@
+package org.opensearch.dataprepper.plugins.source.sqs;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.Max;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import java.time.Duration;
+
+import org.hibernate.validator.constraints.time.DurationMax;
+import org.hibernate.validator.constraints.time.DurationMin;
+
+public class QueueConfig {
+
+    private static final Integer DEFAULT_MAXIMUM_MESSAGES = null;
+    private static final Boolean DEFAULT_VISIBILITY_DUPLICATE_PROTECTION = false;
```

Comment:
S3 source current defaults to not using this, and i think this is more significant in the S3 case since a single message can refer to a large s3 object and hence taking a lot of time. 

On the contrary, I don't see a reason not to use this given the benefits